src/main/java/com/easyrest/constants/Constants.java

```diff
@@ -6,4 +6,12 @@ public static class TimeoutVariable {
         public static final int IMPLICIT_WAIT = 4;
         public static final int EXPLICIT_WAIT = 7;
     }
+
+    public static class AdministratorTabMenu {
```

Comment:
will you use this constant in one class or in several classes?

Reply:
I use this constant in one class for readability and store them in a constants package for maintainability.